/var/spool/slurmd/job1440778/slurm_script: line 20: /hkfs/home/project/hk-project-test-p0021478/cc7738/anaconda3/etc/profile.d/conda.sh: Permission denied
/var/spool/slurmd/job1440778/slurm_script: line 22: conda: command not found
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
Traceback (most recent call last):
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from .storage import SparseStorage  # noqa
    from .storage import SparseStorage  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
    from .storage import SparseStorage  # noqa
    from .storage import SparseStorage  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
    from torch_sparse.utils import Final, index_sort
    from torch_sparse.utils import Final, index_sort
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
    from torch_sparse.utils import Final, index_sort
    from torch_sparse.utils import Final, index_sort
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
    import torch_sparse.typing
    import torch_sparse.typing
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
    import torch_sparse.typing
    import torch_sparse.typing
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
    import pyg_lib  # noqa
    import pyg_lib  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
    import pyg_lib  # noqa
    import pyg_lib  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
    load_library('libpyg')
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    torch.ops.load_library(spec.origin)
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:54: UserWarning: An issue occurred while importing 'pyg-lib'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'pyg-lib'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
  warnings.warn("Please install 'pytorch_lightning' via  "
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from .storage import SparseStorage  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
    from torch_sparse.utils import Final, index_sort
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
    from .storage import SparseStorage  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
    from torch_sparse.utils import Final, index_sort
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
    import torch_sparse.typing
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
    import torch_sparse.typing
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
    import pyg_lib  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from .storage import SparseStorage  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    from torch_sparse.utils import Final, index_sort
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
    import pyg_lib  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
Traceback (most recent call last):
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/gcns_gerrman/test_ns.py", line 15, in <module>
    from graphgps.train.ns_train import Trainer_NS
  File "/hkfs/work/workspace/scratch/cc7738-benchmark_tag/TAPE_gerrman/core/graphgps/train/ns_train.py", line 26, in <module>
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
    import torch_sparse.typing
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
    from torch_sparse import SparseTensor
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/__init__.py", line 39, in <module>
    from .storage import SparseStorage  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/storage.py", line 7, in <module>
    import pyg_lib  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
    from torch_sparse.utils import Final, index_sort
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/utils.py", line 5, in <module>
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    import torch_sparse.typing
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch_sparse/typing.py", line 2, in <module>
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    import pyg_lib  # noqa
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 39, in <module>
    load_library('libpyg')
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/pyg_lib/__init__.py", line 36, in load_library
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
    torch.ops.load_library(spec.origin)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/torch/_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "/home/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /hkfs/home/haicore/aifb/cc7738/anaconda3/envs/EAsF/lib/python3.10/site-packages/libpyg.so)
/var/spool/slurmd/job1440778/slurm_script: line 49: /dev/tty: No such device or address
/var/spool/slurmd/job1440778/slurm_script: line 53: kill: (2321769) - No such process
